src/components/SelectBox/index.tsx

```diff
@@ -55,59 +58,63 @@ const formatCountriesData = (countries:object) =>{
       }
       return 0;
     }
-  )
-}
-
-
+  );
+};
 
-export const SelectCountryRegionBox:React.FunctionComponent<SelectBoxProps> =  props =>{
-  const { fields, updateField, location } = props
-  const label = location === "origin" ? "Origin" : "Destination"
-  const countriesData = formatCountriesData(countries)
+export const SelectCountryRegionBox: React.FunctionComponent<SelectBoxProps> =  (props) => {
+  const { fields, updateField, location } = props;
+  const label = location === 'origin' ? 'Origin' : 'Destination';
+  const countriesData = formatCountriesData(countries);
   const regionsData = React.useMemo(
-    ()=> function (name:string):Array<string> {
-      return countriesData.find(
-        country=>(country.name == name)
-      ).regions
-      }(fields['country']), [fields['country']]
-  )
+    () => {
+      // tslint:disable-next-line:prefer-array-literal
+      return function (name: string): Array<string> {
+        return countriesData.find(
+          country => (country.name === name)
+        ).regions;
+      }(fields.country);
+    }, [fields.country]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [arguments are not aligned](https://app.codacy.com/app/mashafrancis/guacamole-fe/pullRequest?prid=4440700)